src/pages/hegel/reference/mechanical-object/section-3.mdx

```diff
@@ -3,20 +3,21 @@ title: Section 3
 description: Learn about the third section of the Mechanical Object
 ---
 
-In Section 3 of “A. The Mechanical Object”, Hegel builds on the analysis of the infinite regress of determinism, because, as Hegel will go on to show, determinism leads to a contradiction that must be resolved. It is this contradiction that will lead us to the next chapter of Mechanism.
-	 We have already said that there is no moment of self-determination within the mechanical objects. Instead, the determinateness that lies between them is entirely external and, owing to their indifference, entirely indeterminate. As such, the determinateness that lies between each mechanical object is identical to every other. To clarify, the determinateness is identical because there is no specific determination given to the determinateness by a particular mechanical object. If every mechanical object is entirely indifferent to every other mechanical object, then the determinateness that lies between them is simply that: the external determinateness between two indifferent and indeterminate mechanical objects. That determinateness is identical for each mechanical object. So whether you have a rock slamming into a rock, or a rock falling into the water, there is no determinate difference. In both cases, we are basically talking about an external determinateness between two indifferent and indeterminate mechanical objects. This is what is going on in the first half of Section 3: 
+In Section 3 of `A. The Mechanical Object`, Hegel builds on the analysis of the infinite regress of determinism, because, as Hegel will go on to show, determinism leads to a contradiction that must be resolved. It is this contradiction that will lead us to the next chapter of Mechanism.
```

Comment:
Empty line needed to separate paragraphs.

Reply:
done